.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+      - name: Set PY
+        run: echo "::set-env name=PY::$(python -VV | sha256sum | cut -d' ' -f1)"
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: Install tox
+        run: pip install tox
+      - name: Test ${{ matrix.toxenv }}
+        run: tox -e ${{ matrix.toxenv }}
+
+  cpython:
+    name: Test
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os:
+          - Ubuntu
+          - Windows
+        python-version:
+          - 2.7
+          - 3.5
+          - 3.6
+          - 3.7
+          - 3.8
+        pip-version:
+          - "20.0"
+          - "latest"
```

Comment:
The exclusions would be
```suggestion
          - "latest"
        exclude:
        - os: Windows
          python-version: pypy
        - os: Windows
          python-version: pypy3
        - pip-version: "20.0"
          python-version: pypy
        - pip-version: "20.0"
          python-version: pypy3
```
and this'll allow to consolidate jobs

Reply:
Yeah, this is much better. Thanks!

Reply:
The key thing that is not very well documented/understood is that you don't have to list all the factors in your exclusions which allows you to have fewer lines of them.

Reply:
<img width="249" alt="Screen Shot 2020-05-02 at 10 44 40 PM" src="https://user-images.githubusercontent.com/578543/80891771-8de17e00-8cc6-11ea-8f24-e9a351597410.png">

Reply:
Yeah, Include and exclude syntax are too unclear.

Reply:
Thanks for the suggestion! Addressed in 988dfbc.